pkg/v1/tkg/test/tkgpackageclient/package_plugin_integration_test.go

```diff
@@ -411,6 +413,7 @@ func testHelper() {
 	err = json.Unmarshal(resultImgPullSecret.Stdout.Bytes(), &imgPullSecretOutput)
 	Expect(err).ToNot(HaveOccurred())
 	Expect(len(imgPullSecretOutput)).To(BeNumerically(">=", 1))
+	 */
```

Comment:
Did this changed pass locally? If not and if the pipeline failed, I'd probably comment till the end of line 437, so that it starts off by adding a repository.